civis/ml/_model.py

```diff
@@ -727,7 +715,7 @@ def from_existing(cls, train_job_id, train_run_id='latest', client=None):
         dependencies = args.get('DEPENDENCIES', None)
         if dependencies:
             dependencies = dependencies.split()
-        git_token_name = args.get('GITHUB')
+        git_token_name = client.credentials.get(args.get('GIT_CRED')).name
```

Comment:
This errors if the target script either doesn't have a "GIT_CRED" or doesn't have the argument filled.